### PR TITLE
Gives ERT Combat Gloves

### PR DESCRIPTION
**Pull request by ppi13**
## What Does This PR Do

Gives all ERT combat gloves by default.

## Why It's Good For The Game

ERT is supposed to deal with emergency situation, and the fact a singly shocky door is kinda silly, getting onto station and instantly running into a shocked door is honestly bullshit.

## Changelog
:cl: ppi
add: Gives all ERT combat gloves by default
/:cl:

### CODE REVIEW

**Comment by Feasel** [comment]
I've seen too many rounds of ERT barely arriving and just instantly getting gamer'd on a shocked door. It's kinda ridiculous to be honest. 

Hell, I've already had a round as Captain where all my ERT was instantly killed because antags shocked arrival doors to kill them all. 

10/10 change.

**Comment by Sirryan2002** [comment]
Needed change, there's no IC reason why they shouldn't have gloves with shock protection.

**Review by S34NW — APPROVED** [review]

**Comment by procdrone** [comment]
Blueshield have combat gloves too, see no reason why ERT can't have em too. Cheers for insuls

**Comment by E-MonaRhg** [comment]
I feel like the medical ERTs should have a pair of nitrile gloves in their backpacks, even though that the combat gloves are almost as good as nitrile when it comes to surgery and all that.

**Review by Necaladun — APPROVED** [review]
as hilarious as killing the entire team because you shocked the shuttle door is...yeah combat gloves makes sense to me.

File: code/modules/response_team/ert_outfits.dm [hunk 1]
```diff
@@ -37,6 +37,7 @@
 	rt_mob_job = "ERT Commander"
 
 	uniform = /obj/item/clothing/under/rank/centcom_officer/sensor
+	gloves = /obj/item/clothing/gloves/combat
```
Comment (AffectedArc07) [inline code/modules/response_team/ert_outfits.dm:40]: Why not just put this on the `/datum/outfit/job/centcom/response_team` level?

Comment (ppi13) [reply code/modules/response_team/ert_outfits.dm]: Mainly cause I did not think of it, since that datum was in it's separate file, but it is done now.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**